src/sst/core/configGraph.cc

```diff
@@ -743,6 +813,19 @@ const ConfigComponent* ConfigGraph::findComponent(ComponentId_t id) const
     return comps[COMPONENT_ID_MASK(id)].findSubComponent(id);
 }
 
+ConfigStatistic* ConfigGraph::findStatistic(StatisticId_t id)
+{
+    return const_cast<ConfigStatistic*>(const_cast<const ConfigGraph*>(this)->findStatistic(id));
+}
+
+const ConfigStatistic* ConfigGraph::findStatistic(StatisticId_t id) const
```

Comment:
Same comment here as above. I would return a non-const `ConfigStatistic*` and not use any const casts

Reply:
👍 modified here : 96555ecf2df86e405e57505ca43a4895865108f7